### DISCUSSION TITLE
Auto launch a package

### DISCUSSION THREAD

**Discussion by Iktwo** [LykosAI/StabilityMatrix#239] [Q&A]
Stability Matrix is awesome. I couldn't find this question/request, maybe I searched in the wrong way, but is there a mechanism to auto start a package?  I have both ComfyUI and Automatic1111, I would like to be able to automatically launch one of the 2 when I open Stability Matrix. Is that doable?

**Comment by StefanKirste** [comment]
or even more.... start webui at systemstart

**Comment by vin901** [comment]
Also interested in this, if anyone knows any command line that can be run to start up the packages then it would be easy to add that as a .BAT file or script that runs at windows startup.

**Reply by vin901** [reply]
Ok searched the code and found the command line arguments in "StabilityMatrix/StabilityMatrix.Avalonia/Models/AppArgs.cs", one of them being:

[Option("launch-package", HelpText = "Package ID or name to launch on startup")]

So you would run StabilityMatrix.exe --launch-package <package_id_OR_name>

**Comment by mohnjiles** [comment]
You can use the `--launch-package` argument to launch SM and start a package at the same time. It uses the display name of the package. You can also add this to a shortcut and put it in your startup folder if you want it to run on startup, like so:
![image](https://github.com/user-attachments/assets/872ffb59-2c63-46f6-a07d-0a2d3b1d4989)

**Reply by vin901** [reply]
@mohnjiles Oh just saw this after posting my own reply haha, thank you sir!
It's useful but I guess you can't start multiple packages?
